src/pages/CreateFeedback.tsx

```diff
@@ -1,206 +1,105 @@
 import { useState } from 'react';
 import { ProductRequest } from '../types';
-import { Link, useNavigate } from 'react-router-dom';
+import { useNavigate, Link } from 'react-router-dom';
 import arrowDown from '../assets/shared/icon-arrow-down.svg';
 import arrowUp from '../assets/shared/icon-arrow-up.svg';
 import leftArrow from '../assets/shared/icon-arrow-left.svg';
 import newFeedback from '../assets/shared/icon-new-feedback.svg';
-
+import { useForm, SubmitHandler } from 'react-hook-form';
 interface IProps {
   productRequests: ProductRequest[];
 }
 
+type Inputs = {
```

Comment:
A couple of "best-practice" naming conventions:

1. Always keep things singular if possible. It's more readable and we can add more types as needed.
2. We want to be more descriptive here as far as the name goes.

Consider this:
```
type CreateFeedbackForm = {
  titleRequired: string;
  feedbackRequired: string;
  category: string;
}
```

Reply:
What suggestions would you have for more descriptive naming. `createFeedback, createCategory, createTitle`?

Reply:
Keeping these as simple as possible is best. So for this,

`feedback, category, title` would be recommended.

These are all driven by the `name` attribute of the html5 input. This would be akin to what would be consumed by an API, so you'd want to ask.. if you were consuming this JSON, what would that structure be? I don't think the response key would be `titleRequired` for example. That's how I came to my suggestion.

Reply:
Oooh ok got it. Updated!